tests/setupTestsAfterEnv.ts

```diff
@@ -0,0 +1,18 @@
+import path from 'path';
+import { setupPolly } from 'setup-polly-jest';
+import NodeHttpAdapter from '@pollyjs/adapter-node-http';
+import FSPersister from '@pollyjs/persister-fs';
+
+global.pollyContext = setupPolly({
```

Comment:
This is a bit of a hack to set up Polly globally and make it accessible in any test. It is a variation on these instructions: https://netflix.github.io/pollyjs/#/test-frameworks/jest-jasmine